pkgs/development/tools/misc/nrf-command-line-tools/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, system
+, udev
+, libusb
+, segger-jlink
+}:
+
+let
+  supported = {
+    x86_64-linux = {
+      name = "linux-amd64";
+      sha256 = "0e036afa51c83de7824ef75d34e165ed55efc486697b8ff105639644bce988e5";
+    };
+    i686-linux = {
+      name = "linux-i386";
+      sha256 = "ba208559ae1195a0d4342374a0eb79697d31d6b848d180ac906494f17f56623b";
+    };
+    aarch64-linux = {
+      name = "linux-arm64";
+      sha256 = "cffa4b8becdb5545705fd138422c648d809b520b7bc6c77b8b50aa1f79ebe845";
+    };
+    armv7l-linux = {
+      name = "linux-armhf";
+      sha256 = "c58d330152ae1ef588a5ee1d93777e18b341d4f6a2754642b0ddd41821050a3a";
+    };
```

Comment:
32 bit OS's don't have good support in NixOS.

Reply:
Should I remove the support then?

Reply:
I'll leave it in then.